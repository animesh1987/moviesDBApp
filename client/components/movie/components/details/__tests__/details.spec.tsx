import * as React from 'react';
import { create } from 'react-test-renderer';
import { shallow } from 'enzyme';

import { MovieDetails } from '../index';

describe('Movie Details Test', () => {
  const title = 'Test';
  const releaseDate = '2017-02-02';

  it('renders correctly', () => {
    const component = create(
      <MovieDetails title={title} release_date={releaseDate} />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
