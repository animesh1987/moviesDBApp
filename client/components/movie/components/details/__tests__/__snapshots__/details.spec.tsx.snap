// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Details Test renders correctly 1`] = `
<div
  className="layout-row layout-align-space-between layout-align-center"
>
  <span
    className="movie-title"
  >
    Test
  </span>
  <span
    className="release-date"
  >
    2017
  </span>
</div>
`;
